src/libraries/Common/src/Interop/BSD/System.Native/Interop.Sysctl.cs

```diff
@@ -30,12 +30,13 @@ internal static unsafe void Sysctl(Span<int> name, ref byte* value, ref int len)
             }
         }
 
+
         private static unsafe void Sysctl(int* name, int name_len, ref byte* value, ref int len)
         {
             IntPtr bytesLength = (IntPtr)len;
             int ret = -1;
             bool autoSize = (value == null && len == 0);
-
+#pragma warning disable CA2020 // Prevent behavioral changes
```

Comment:
Where was the behavior change in your change? I did not see any problem with it.

Reply:
Couldn't figure out what causing this:
![image](https://user-images.githubusercontent.com/45579687/191891311-229a8b18-455c-48da-a253-b9122f4812c2.png)
None of the parameters/arguments are `nint` or `void*`, could not repro it locally with whatever I tried, my best guess is  changes in this file might causing this

Reply:
It does not make sense for changes in one file to cause a build error in completely unrelated project.

You need to `git fetch upstream` and `git merge upstream/main` into your branch to see this error locally. There are new changes in main in this file that need some new fixes to build with this change.

Reply:
By the way `Prevent behavioral changes` is the analyzer title, not related to  this change, here just reverting the to the old code and suppressing the warning

Reply:
> You need to git fetch upstream and git merge upstream/main into your branch to see this error locally.

Oh sure, thanks